KeePassKit/Core/KPKTimeInfo.m

```diff
@@ -220,11 +220,13 @@ - (void)_updateExpireState {
 }
 
 - (void)_reducePrecicionToSeconds {
+  self.updateTiming = NO;
```

Comment:
This isn't strictly necessary but thers a macro in KPKScopedSet.h to set something inside a scope.

```
KPK_SCOPED_SET_BEGIN(self.updateTiming);
...
KPK_SCOPED_NO_END(self.updateTiming);
```

Reply:
PR updated